addons/web_dashboard/static/src/less/dashboard.less

```diff
@@ -0,0 +1,136 @@
+@bg-main:#F2F2F2;
+
+.dashboard {
```

Comment:
prefix all your CSS class with `o_web_dashboard_`
Use maximum native bootstrap class. You have too much less.